draft-ietf-masque-connect-ip.md

```diff
@@ -625,54 +714,54 @@ examples that are relevant for CONNECT-IP include:
 In order to receive these errors, endpoints need to be prepared to receive ICMP packets.
 If an endpoint sends ROUTE_ADVERTISEMENT capsules, its routes SHOULD include an allowance
 for receiving ICMP messages. If an endpoint does not send ROUTE_ADVERTISEMENT capsules,
-such as a client opening an IP flow through a proxy, it SHOULD process proxied ICMP packets
+such as a client opening an IP flow through an IP proxy, it SHOULD process proxied ICMP packets
 from its peer in order to receive these errors. Note that ICMP messages can originate from
-a source address different from that of the CONNECT-IP peer.
+a source address different from that of the IP proxying peer.
 
 # Examples
```

Comment:
Just double-checking a minor point (not related to this PR but maybe we can address this here as well if needed at all): Why are we using stream ID 44 in two examples and a different stream ID in the other..? Is there any particular reason? In order to avoid confusion, should we maybe either always use the same or always use a different one?

Reply:
Fixed